### PR TITLE
Improve Snuba config

### PR DESCRIPTION
**Pull request by J0sh0nat0r**
This PR improves the Snuba config in-order to reduce the maintanance required to support future versions of Snuba.

### CODE REVIEW

File: sentry/templates/configmap-snuba.yaml [hunk 1]
```diff
@@ -11,36 +11,28 @@ metadata:
...
+    CLUSTERS[0]["single_node"] = env("CLICKHOUSE_SINGLE_NODE", "false").lower() == "true"
 
+    # Redis Options
     REDIS_HOST = {{ include "sentry.redis.host" . | quote }}
     REDIS_PORT = {{ include "sentry.redis.port" . }}
     {{- if $redisPass }}
     REDIS_PASSWORD = {{ $redisPass | quote }}
     {{- end }}
     REDIS_DB = int(env("REDIS_DB", 1))
-    USE_REDIS_CLUSTER = False
```
Comment (Mokto) [inline sentry/templates/configmap-snuba.yaml:28]: You removed this and both DOGSTATSD, I just want to make sure it's on purpose?

Comment (J0sh0nat0r) [reply sentry/templates/configmap-snuba.yaml:28]: Yes, we should now be able to inherit the values from the default `settings.py`

**Review by Mokto — APPROVED** [review]

File: sentry/templates/configmap-snuba.yaml (hunk 1 above)
Comment (Mokto) [reply sentry/templates/configmap-snuba.yaml:28]: 👍

File: sentry/templates/configmap-snuba.yaml [hunk 2]
```diff
@@ -11,36 +11,28 @@ metadata:
...
-            "storage_sets": {"events", "events_ro", "migrations", "outcomes", "querylog", "sessions", "transactions"},
-            # FIXME: Snuba will be able to migrate multi node clusters in the future
-            "single_node": env("CLICKHOUSE_SINGLE_NODE", "false").lower() == "true"
-        },
-    ]
+    # Processor/Writer Options
+    DEFAULT_BROKERS = [{{ printf "%s:%s" (include "sentry.kafka.host" .) (include "sentry.kafka.port" .) | quote }}]
 
-{{ .Values.config.snubaSettingsPy | indent 4 }}
+    {{ .Values.config.snubaSettingsPy | indent 4 }}
```
Comment (yoan-adfinis) [inline sentry/templates/configmap-snuba.yaml]: this is likely to break (unless.

Comment (yoan-adfinis) [reply sentry/templates/configmap-snuba.yaml]: .. it starts with a comment)

Comment (J0sh0nat0r) [reply sentry/templates/configmap-snuba.yaml]: Whoops, yeah

**Comment by J0sh0nat0r** [comment]
@Mokto Hey just wondering if you wanted me to merge this or if something's blocking?

**Comment by Mokto** [comment]
Go for it!

**Pull request merged**